Makefile

```diff
@@ -50,7 +47,4 @@ check-latest-imagestream:
 check-betka:
 	cd tests && ./check_betka.sh
 
-check-remote-containers:
-	TESTED_IMAGES="$(TESTED_IMAGES)" tests/remote-containers.sh
-
-check: check-failures check-squash check-latest-imagestream check-remote-containers
+check: check-failures check-squash check-latest-imagestream
```

Comment:
Empty `Makefile` targets caused us some problems before. Could you please remove the target entirely or add some content to it?

Reply:
Good point. I have refactored it a bit